exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestInfoSchema.java

```diff
@@ -64,13 +64,13 @@ public static void setupFiles() {
 
   @Test
   public void selectFromAllTables() throws Exception{
-    test("select * from INFORMATION_SCHEMA.SCHEMATA");
-    test("select * from INFORMATION_SCHEMA.CATALOGS");
-    test("select * from INFORMATION_SCHEMA.VIEWS");
-    test("select * from INFORMATION_SCHEMA.`TABLES`");
-    test("select * from INFORMATION_SCHEMA.COLUMNS");
-    test("select * from INFORMATION_SCHEMA.`FILES`");
-    test("select * from INFORMATION_SCHEMA.`PARTITIONS`");
+//    test("select * from INFORMATION_SCHEMA.SCHEMATA");
+//    test("select * from INFORMATION_SCHEMA.CATALOGS");
+//    test("select * from INFORMATION_SCHEMA.VIEWS");
+    test("select * from INFORMATION_SCHEMA.`TABLES` where table_schema = 'cp.default'");
+//    test("select * from INFORMATION_SCHEMA.COLUMNS");
+//    test("select * from INFORMATION_SCHEMA.`FILES`");
+//    test("select * from INFORMATION_SCHEMA.`PARTITIONS`");
```

Comment:
Please add a new test instead of changing the existing one.

Reply:
@vvysotskyi thanks, this was definitely not supposed to be included in what I pushed.  Reverted this test, another new one is already present.